### DISCUSSION TITLE
The log is stuck to the previous one.

### DISCUSSION THREAD

**Discussion by edgar1170** [qos-ch/logback#638] [General]
![image](https://user-images.githubusercontent.com/117433297/226794017-1f1ab05c-bbe5-4430-ba97-d10a6cf1b1c8.png)
<dependency>
            <groupId>ch.qos.logback</groupId>
            <artifactId>logback-core</artifactId>
            <version>1.2.9</version>
</dependency>
Can I solve the problem by upgrading the version?

**Comment by ceki** [comment]
If I understand correctly, there is no linefeed separator between to logging events. Can you share your logback.xml configuration?

**Reply by edgar1170** [reply]
%msg%n in encoder.pattern
 <encoder>
            <pattern>%msg%n</pattern>
        </encoder>

**Reply by edgar1170** [reply]
![image](https://user-images.githubusercontent.com/117433297/233818731-1b3f0095-0ac6-4ad1-a4db-abe5e44249b7.png)

**Comment by edgar1170** [comment]
Who can help me?

**Comment by ceki** [comment]
@edgar1170 Is this issue being observed regularly or seldom?

**Reply by edgar1170** [reply]
seldom

**Comment by edgar1170** [comment]
?

**Comment by ceki** [comment]
This problem has never been reported previously. Can you provide a test reproducing the problem? Otherwise, please close this discussion.
